CONTRIBUTING.md

```diff
@@ -12,12 +12,12 @@ document and take heed of its wisdom.
 
 ## Pull Requests
 
-This project uses Travis to check that it always builds on all the supported
-platforms. However, Microsemi's IT department stubbornly refuses (for silly
-reasons) to turn it on for the main repository. Therefore, in order to ensure
-your contributions get tested, please submit all pull requests to
-[lsgunth/switchtec-user][2]. From there we will ensure your contribution
-gets merged into the main repository.
+Please set the base branch to microsemi/devel when create the pull request.
+Make sure the comparing changes inclue your bug fixes or new features, the
+patches has no obvious style problems and is ready for submission when
+check by checkpatch.pl script of the kernel tree.
```

Comment:
You should say script of the "Linux Kernel tree".

Just "kernel" is ambiguous.